src/redux/action.ts

```diff
@@ -17,13 +17,23 @@ import {
     FilterKeys,
     SortKeys
 } from './enums';
+import {Movie} from './types';
 
 interface Params {
     filter?: FilterKeys;
     sort?: SortKeys;
     search?: string;
 }
 
+export const dispatchMovies = (
```

Comment:
IDK if this is suitable name for this function. You've already have **getMovies**, maybe **setMovies** will be better here?